src/components/OverviewPage/TransactionList.tsx

```diff
@@ -61,14 +61,11 @@ const OverviewPageTransactionList = ({ className, onTransactionClick }: Overview
 
   return (
     <Table isLoading={showSkeletonLoading} className={className} minWidth="500px">
-      {allPendingTxs
-        .slice(0)
-        .reverse()
```

Comment:
I don't know why this was like this, pending addresses should be displayed like any other.